phpunit

```diff
@@ -1,38 +1,12 @@
 #!/usr/bin/env php
 <?php
-/* PHPUnit
+/*
+ * This file is part of the PHPUnit.
```

Comment:
Agreed, I will change as soon as I can

On Wed, Dec 31, 2014, 22:33 Graham Campbell <notifications@github.com>
wrote:

> In phpunit
> <https://github.com/sebastianbergmann/phpunit/pull/1558#discussion-diff-22397072>
> :
>
> > @@ -1,38 +1,12 @@
> >  #!/usr/bin/env php
> >  <?php
> > -/* PHPUnit
> > +/*
> > + * This file is part of the PHPUnit.
>
> Surely This file is part of PHPUnit. rather than This file is part of the
> PHPUnit.?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/sebastianbergmann/phpunit/pull/1558/files#r22397072>.
>